Builds/VisualStudio2017/README.md

```diff
@@ -0,0 +1,260 @@
+need these software components:
+
+* [Visual Studio 2017](README.md#install-visual-studio-2017)
+* [Git for Windows](README.md#install-git-for-windows)
+* [Google Protocol Buffers
+  Compiler](README.md#install-google-protocol-buffers-compiler)
+* [OpenSSL Library](README.md#install-openssl)
+* [Boost library](README.md#build-boost)
+* (Optional) [Cmake for Windows](README.md#optional-install-cmake-for-windows) -
+  Only needed if not using the integrated CMake in VS 2017.
+
+## Install Software
+
+### Install Visual Studio 2017
+
+If not already installed on your system, download your choice of installer from
+the [Visual Studio 2017
+Download](https://www.visualstudio.com/downloads/download-visual-studio-vs)
+page, run the installer, and follow the directions. **You may need to choose the
+`Desktop development with C++` workload to install all necessary C++ features.**
+
+Any version of Visual Studio 2017 may be used to build rippled. The **Visual
+Studio 2017 Community** edition is available free of charge (see [the product
+page](https://www.visualstudio.com/products/visual-studio-community-vs) for
+licensing details), while paid editions may be used for an free initial trial
+period.
+
+As of this writing, the latest version of Visual Studio 2017 for Windows is
+15.5.4.
+
+### Install Git for Windows
+
+Git is a distributed revision control system. The Windows version also provides
+the bash shell and many Windows versions of Unix commands. While there are other
+varieties of Git (such as TortoiseGit, which has a native Windows interface and
+integrates with the Explorer shell), we recommend installing [Git for
+Windows](https://git-scm.com/) since it provides a Unix-like command line
+environment useful for running shell scripts. Use of the bash shell under
+Windows is mandatory for running the unit tests.
+
+As of this writing, the latest version of Git for Windows is 2.16.1.
+
+### Install Google Protocol Buffers Compiler
+
+Building rippled requires **protoc.exe** version 2.5.1 or later. At your option
+you may build it yourself from the sources in the [Google Protocol
+Buffers](https://github.com/google/protobuf) repository, or you may download a
+[protoc.exe](https://ripple.github.io/Downloads/protoc/2.5.1/protoc.exe)
+([alternate
+link](https://github.com/ripple/Downloads/raw/gh-pages/protoc/2.5.1/protoc.exe))
+precompiled Windows executable from the [Ripple
+Organization](https://github.com/ripple).
+
+Either way, once you have the required version of **protoc.exe**, copy it into a
+standard location that is in your command line `%PATH%`.
+
+* **NOTE:** If you use an older version of the compiler, the build will fail
+  with errors related to a mismatch of the version of protocol buffer headers
+  versus the compiler. Likewise, if you use version 3 or newer, the build will
+  fail.
+
+### Install OpenSSL
+
+[Download OpenSSL.](http://slproweb.com/products/Win32OpenSSL.html) There will
+be four variants available:
```

Comment:
You mention four variants then only list two. There will also be a link to download openssl 1.1, and people may be tempted to use it. However, rippled does not compile with openssl 1.1, only the 1.0 branch (at least last I checked, is this still true?)

Reply:
That is still true AFAIK. I'll fix.